docs/resources/deployment_traffic_filter_association.md

```diff
@@ -8,12 +8,9 @@ description: |-
 
 Provides an Elastic Cloud traffic filter association resource, which allows traffic filter rules to be associated to an Elastic Cloud deployment outside of the control of Terraform. Associations can be created and deleted.
 
```

Comment:
The same comment about the deleted note as the one for `traffic_filter` attribute of `deployment` resource.